R/biomass.R

```diff
@@ -63,7 +63,7 @@ Zoopbiomass<-function(Zoop,
       dplyr::left_join(ZoopLengths%>%
                          dplyr::left_join(Biomass_macro, by="Taxname")%>%
                          dplyr::filter(!is.na(.data$a))%>%
-                         dplyr::mutate(Mass=0.1*0.4*(.data$a*.data$Length^.data$b)*.data$Count)%>%
+                         dplyr::mutate(Mass=0.25*0.4*(.data$a*.data$Length^.data$b)*.data$Count)%>%
```

Comment:
Sorry I might have missed it but can you describe the basis for this change? It would be good to also add this explanation to the news file

Reply:
Christina and I have been fighting with length-weight regressions for the past two months. This is, frankly, the only thing that makes any sense. Otherwise the weights calculated from formulas using dry mass end up being WAY bigger than the ones that were calculated using wet mass, even when they were based on the same species. According to this dataset, dry-to-wet ratios vary by taxonomic group from 0.16 to 0.26, with amphipods at 0.26. We only use wet-weight equations for the amphipods, so it seems like 0.25 would be more appropriate.

But in the process of writing this reply I notice that the original code only used the dry weight formalin length-weight regressions. Which means the final weight should have only been multiplied by 0.4 (to convert dry weight to carbon weight). Did you intend to only calculate mass for the two taxa that have dry weight, formalin regressions?

Reply:
yeah I think I intended to only use the two taxa that have dry weight regressions to be comparable with the other taxa, and only formalin since EMP preserves in formalin. I'm not sure I was aware of a dry to wet weight conversion factor. I really should have documented this better, but I think the 0.1 in this equation was meant to convert the units from mg to micrograms but if that's the case, it should have been 0.001. Do the macro zoop biomass numbers look 100X higher than they should be to you?

Reply:
I haven't actually used the Zoopbiomass function yet, but, the current documentation for the Zoopbiomass function doesn't say what the final units are. If you drill down into the biomass_macro table, it says the units are mg. 

If we are converting to carbon weight rather than just dry weight, that should probably be in the documentation, and I think it would make sense for the function to give the same units for both Macro and Mesomicro.

Do you want me to make that change?

Reply:
yeah I can fix the documentation, I'll wait until this PR is merged though to avoid github issues. Can you just revert your change here?

Reply:
Sure!

Reply:
If we want this to work with more species we could allow the wet weight equations using your conversion factor, although it would be much better if someone would make some dry weight length-weight equations for the remaining species.